<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSender</class>
 <widget class="QWidget" name="FileSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileSender</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_sender">
      <attribute name="title">
       <string>Sender</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_add_send_user">
                 <property name="text">
                  <string>Choose User</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_add_send_user">
                 <property name="text">
                  <string>Choose</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListView" name="list_send_user"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_add_send_file">
                 <property name="text">
                  <string>Choose Send File[s]</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_add_send_file">
                 <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                 </property>
                 <property name="text">
                  <string>Choose File[s]</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListView" name="list_add_send_file"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10,1">
           <item>
            <widget class="QLabel" name="label_send_status">
             <property name="text">
              <string>Send Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="process_send">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_send_to_user">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_recv">
      <attribute name="title">
       <string>Reciever</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_sending_user">
               <property name="text">
                <string>Sending User</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="list_sending_user"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_sended_file">
               <property name="text">
                <string>Recieved File[s]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="list_sending_file"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,10,1">
           <item>
            <widget class="QLabel" name="label_recv_status">
             <property name="text">
              <string>Recieve Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="process_recv">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_recv_save_file">
             <property name="text">
              <string>Add File Path</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_add_user">
      <attribute name="title">
       <string>Add User</string>
      </attribute>
      <widget class="QPushButton" name="btn_add_user">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>141</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>Add User</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_accept_ok">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>200</y>
         <width>141</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_is_accept">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Accept???</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_add_user">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose User[s] to Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_accept_no">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>200</y>
         <width>141</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>No Accepting</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
